backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -249,13 +252,18 @@ def fetch_response(self, request: HttpRequest, pk: Any = None) -> Response:
         tool_id: str = str(custom_tool.tool_id)
         document_id: str = request.data.get(ToolStudioPromptKeys.DOCUMENT_ID)
         id: str = request.data.get(ToolStudioPromptKeys.ID)
+        run_id: str = request.data.get(ToolStudioPromptKeys.RUN_ID)
+        if not run_id:
+            # Generate a run_id
+            run_id = str(uuid.uuid4())
```

Comment:
@gaya3-zipstack Let's introduce a utility function in prompt studio helper to generate a run ID and use this everywhere instead.

Reply:
Done